src/Microsoft.DotNet.TestFramework/Microsoft.DotNet.TestFramework.csproj

```diff
@@ -14,18 +14,15 @@
     <Compile Include="**\*.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.DotNet.Cli.Utils\Microsoft.DotNet.Cli.Utils.csproj" />
+    <ProjectReference Include="../Microsoft.DotNet.Cli.Utils\Microsoft.DotNet.Cli.Utils.csproj" />
```

Comment:
I thought we were going with backslashes for MSBuild. Either way is fine. Just need confirmation of what it is going to be.